Gemfile

```diff
@@ -43,6 +43,7 @@ end
 group :development do
   gem 'annotate', '~> 3.0', '>= 3.0.3'
   gem 'better_errors', '~> 2.5', '>= 2.5.1'
+  gem 'binding_of_caller', '~> 0.8.0'
```

Comment:
why is this needed?

Reply:
It's just so we can use the browser console described here https://github.com/rootstrap/rails_api_base/pull/211#issuecomment-634264205